src/picongpu/include/simulationControl/MovingWindow.hpp

```diff
@@ -220,8 +220,8 @@ class MovingWindow
         const SubGrid<simDim>& subGrid = Environment<simDim>::get().SubGrid();
         Window window;
 
-        window.localDimensions = Selection<simDim>(subGrid.getLocalDomain().size);
-        window.globalDimensions = Selection<simDim>(subGrid.getGlobalDomain().size);
+        window.localDimensions = Selection<simDim>(subGrid.getLocalDomain());
+        window.globalDimensions = Selection<simDim>(subGrid.getGlobalDomain());
```

Comment:
keep, `.size`; offset of *global* window is equal to `offsetFirstGPU` an all GPUs